Examples/Swift/ViewController.swift

```diff
@@ -265,6 +272,20 @@ class ViewController: UIViewController, MGLMapViewDelegate {
             self.mapView?.showWaypoints(currentRoute)
         }
     }
+    
+    func mapViewDidFinishLoadingMap(_ mapView: MGLMapView) {
+        // Demonstrate adding an interactive annotation to the map in NavigationViewController.
+        if let coordinate = routes?.first?.legs.first?.steps.first?.coordinates?.last {
+            let annotation = MGLPointAnnotation()
+            annotation.coordinate = coordinate
+            annotation.title = "End of First Step"
+            mapView.addAnnotation(annotation)
+        }
+    }
+    
+    func mapView(_ mapView: MGLMapView, annotationCanShowCallout annotation: MGLAnnotation) -> Bool {
+        return mapView != self.mapView
```

Comment:
`ViewController.mapView` is the application’s main map. I don’t care to make annotations show callouts there because that doesn’t demonstrate the changes in this PR. Otherwise, `mapView` can be the map view inside NavigationViewController, which I do want showing callouts. In any case, I’m going to remove this example code because it clutters up the application; the navigation-ios-examples repository is a better home for it.